### PR TITLE
:robot: Use earthly

### PR DESCRIPTION
**Pull request by Itxaka**
Changes to use earthly instead of a Makefile.

As a side effect, it shows that some of the tests were not correctly done and they were behaving differently due to the environment.

Plus as a side effect, becuase we are running as root in the earthly env, we can test the full suite and drop the tests for non-root that made no sense

### CODE REVIEW

File: pkg/utils/utils_test.go [hunk 1]
```diff
@@ -690,12 +688,6 @@ var _ = Describe("Utils", Label("utils"), func() {
 			Expect(err).ShouldNot(HaveOccurred())
 			Expect(size).To(Equal(int64(3072)))
 		})
-		It("Returns the expected size of a test folder", func() {
```
Comment (mauromorales) [inline pkg/utils/utils_test.go:693]: why does this one need to be removed?

File: pkg/luet/luet_test.go [hunk 2]
```diff
@@ -72,11 +72,6 @@ var _ = Describe("Types", Label("luet", "types"), func() {
 		cleanup()
 	})
 	Describe("Luet", func() {
-		It("Fails to unpack without root privileges", Label("unpack"), func() {
```
[the remark this answers is not in the document]
Comment (Itxaka) [reply pkg/luet/luet_test.go:75]: This was a shortcoming of running the tests under non root user, we kept adding test to test for non root failures which IMO it's a waste of test. Afterwards, root tests were added and as a consequence, the proper tests were added.

Now that we can run all tests under root they make no sense and splitting or adding more complexity to check that indeed they fail when user is no root is pointless. After all the commands already check if you are running under root and will exit with an error if you are not root

File: pkg/utils/utils_test.go (hunk 1 above)
Comment (Itxaka) [reply pkg/utils/utils_test.go:693]: This was using the non root path to somehow check for a failure getting the size of a dir, which is absurd. This is basically checking Linux permissions lol

Comment (mauromorales) [reply pkg/utils/utils_test.go:693]: gotcha

File: pkg/luet/luet_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (mauromorales) [reply pkg/luet/luet_test.go:75]: yup makes sense

**Review by mauromorales — APPROVED** [review]
lgtm

**Pull request merged**